### ISSUE TITLE
Can't copy tweaked recipes?

### ISSUE THREAD

**Issue by G-Stav** [codetaylor/artisan-worktables#167]
Issue Description
There is a strong likelihood that I'm just doing something wrong, but I'm attempting to copy some tweaked recipes.

When attempting to copy a custom recipe by name, it fails. And when attempting to copy it by item output, it copies the original recipe instead of the tweaked recipe. 

```
import crafttweaker.oredict.IOreDictEntry;
import crafttweaker.item.IItemStack;
import mods.artisanworktables.builder.RecipeBuilder;
import mods.artisanworktables.builder.Copy;

<ore:stickToola>.add(<betterwithmods:material:36>);

recipes.addShaped("shovelaluminuma", <thermalfoundation:tool.shovel_aluminum>,
	[[null, <ore:ingotAluminium>, null],
	[null, <ore:stickToola>, null],
	[null, <ore:stickToola>, null]]);

RecipeBuilder.get("basic")
  .setCopy(Copy.byName("crafttweaker:shovelaluminiuma"))
  .addTool(<ore:artisansHammer>, 10)
  .create();
```

Crash Log
Game did not crash. Please let me know if I should force one for diagnostics.
Affected Versions

    Minecraft: 1.12.2
    Forge: 14.23.5.2836
    CraftTweaker: 4.1.19
    Artisan Worktables: 1.21.4
    Athenaeum: 1.17.1

**Comment by codetaylor** [comment]
Currently, Artisan Worktables (AW) does not support what you are trying to do.

It will only copy recipes from the vanilla crafting recipe registry that exist before CrT resolves.
It is set up this way to allow recipes to be copied by AW before being removed by CrT.

I may look into enabling a way to do this in the future. It could be a few weeks or more before I can even start to look into it, though, so please plan accordingly.

1. Pre-script event: AW resolves copied recipes
2. CrT remove, add
3. Post-script event: *<-- maybe we could introduce a builder flag to indicate a copy directive should resolve here instead*

**Comment by codetaylor** [comment]
A new copy directive `runAfter()` has been added and will be available in version `1.23.0`
